### DISCUSSION TITLE
Welcome to kubectl.nvim Discussions!

### DISCUSSION THREAD

**Discussion by Ramilito** [Ramilito/kubectl.nvim#208] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mosheavni** [comment]
🚀
thanks @Ramilito  for all your effort here

**Comment by paul-enz** [comment]
Hey folks, loving the plugin! Incredible work.

I'm totally open to helping you out, but as a Lua newbie I don't have much of an idea where to help first. I'd appreciate any direction and/or suggestions on how best to help your project (maybe others are too?).

For context, I have the plugin running off a local clone, so I can start making changes. No pressure, just happy to help in any way. If nothing comes to mind, no matter how trivial or documentaton-oriented, no problem! Your work on this is very much appreciated.

**Reply by Ramilito** [reply]
Hi and welcome!
I'm super happy that you like the plugin! 🙏 

I'll create a more technical description on various parts and the architecture design but in the mean time, having a local clone is a super good start!
When it comes to opensource, the most important part is to have fun and do what you want to do...since no one is getting paid 😅.
I suggest finding something you are interested in or a feature you want to have, open a issue and we can discuss the details in the issue. If you want some ideas and such I think you can find some under issues. Documentation can always be better so if there were any parts that were hard to understand or you think we need clarification please expand on that!

What I'm trying to say I think is, feel free to ask questions, create issues and have fun with it!

PS. If it's a feature, it would be wisest to create an issue first so that you don't waste your time if it's something we might not want to support. Some features are awesome but cost a lot of maintenance or some other reason to not implement it.

**Reply by Ramilito** [reply]
Also!
If you find an bug or a weird behaviour, it's super appreciated to just create a issue even if it's vague. 
People underestimate how valuable that is when there are so many variation of things, so many plugin combinations, the order of installing and running plugins, different OS, neovim versions...so many variables that is impossible to test on one/two machines.

**Comment by kjunghoan** [comment]
Started using this a couple of months ago and love it. Keep up the good work

**Reply by Ramilito** [reply]
Thanks for sharing and I agree! It's starting become a great program!
